index.js

```diff
@@ -57,6 +61,32 @@ module.exports.mime = mime
 var listenerCount = EventEmitter.listenerCount
   || function(emitter, type){ return emitter.listeners(type).length; };
 
+
+function responseStatus(res, code, msg) {
+  if(msg == null) {
+    msg = statuses[code]
+
+    res.setHeader('Content-Type', 'text/plain; charset=UTF-8')
+  }
+
+  res.statusCode = code
+  res.setHeader('Content-Length', Buffer.byteLength(msg))
+  res.setHeader('X-Content-Type-Options', 'nosniff')
+  res.end(msg)
+}
+
+function redirect(res, loc) {
+  // redirect
+  res.setHeader('Content-Type', 'text/html; charset=UTF-8')
```

Comment:
This refactored function is missing the `X-Content-Type-Options` security header the old code was setting. Please add it back :)